rts/Game/SyncedGameCommands.cpp

```diff
@@ -567,6 +523,23 @@ class SkipActionExecutor : public ISyncedActionExecutor {
 	}
 };
 
+
+class TakeActionExecutor : public ISyncedActionExecutor {
+public:
+	TakeActionExecutor() : ISyncedActionExecutor(
+		"Take",
+		"Transfers all units of allied teams without any active players to the team of the issuing player"
+	) {}
+
+	bool Execute(const SyncedAction& action) const final {
+		if (luaRules != nullptr) {
+			luaRules->GotChatMsg(action.GetCmd() + " " + action.GetArgs(), action.GetPlayerID());
```

Comment:
I think at this point it may as well be removed since it's equal to just `/luarules take`. If you're worried that players are already used to `/take` then add a trivial widget that listens to `/take` and spits out `/luarules take` in its place.